editor/plugins/curve_editor_plugin.cpp

```diff
@@ -62,15 +61,15 @@ void CurveEdit::set_curve(Ref<Curve> p_curve) {
 	}
 
 	if (curve.is_valid()) {
-		curve->disconnect(CoreStringNames::get_singleton()->changed, callable_mp(this, &CurveEdit::_curve_changed));
-		curve->disconnect(Curve::SIGNAL_RANGE_CHANGED, callable_mp(this, &CurveEdit::_curve_changed));
+		curve->disconnect_changed(callable_mp(this, &CurveEdit::_curve_changed));
+		curve->disconnect_changed(callable_mp(this, &CurveEdit::_curve_changed));
```

Comment:
Is that right?

Reply:
no .-.

Reply:
According to a crude regex, these are the only two

(checked for removed lines with `^-(.*)connect\(("changed"|CoreStringNames|SNAME\("changed")`)